samples/subapp1-online-store/src/footer/subapp-footer.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-magic-numbers */
+
+import { React, loadSubApp } from "subapp-react";
+
+let data;
+
+const Data1 = () => {
+  if (!data) {
+    throw new Promise(resolve => {
+      setTimeout(() => {
+        data = "demo data received from async call";
+        resolve(data);
+      }, 2500);
+    });
+  }
+  return <div>{data}</div>;
```

Comment:
Major: This does not seem to work. I recommend use a similar approach with Timeout and React.Lazy, but ditch reassigning the data1 value.